packages/docs/package.json

```diff
@@ -9,7 +9,9 @@
     "build": "vuepress build docs --temp .temp",
     "view-info": "vuepress view-info docs --temp .temp",
     "show-help": "vuepress --help",
-    "lint-md": "textlint ./docs/**/*.md"
+    "lint-md:wording": "textlint ./docs/**/*.md",
+    "lint-md:style": "remark --quiet --frail .",
+    "lint-md": "pwd"
```

Comment:
Why do we want to print the working directory? The precommit hook currently runs this command so we're no longer going to run the linting commands... unless I'm missing something 😀

Reply:
@sarahdayan oops!